UnrealPlugin/README.md

```diff
@@ -218,16 +206,30 @@ Don't forget to connect all the nodes to the Event Init node.
 In the end add the "Ready for Players" to be able to react to the ready signal of Playfab.
 ![Playfab GSDK Full Graph](Documentation/BlueprintFullGraph.png)
 
+## Set the Game Instance class
+    
+After creating a custom game instance class that integrates with the gsdk, you have to configure your project to actually use this newly created game instance class. There are two ways to do this - either through the Unreal Engine editor or by editing DefaultEngine.ini directly. 
+###### In the Unreal Editor
+In the editor, This can also be set through the UI in the editor. In the editor go to Edit -> Project Settings. From that opened window,
```

Comment:
nit: "This" should start with a lowercase "T"

Reply:
https://github.com/chris-gong/gsdk/commit/18ed3da70e45f01688706bc547c947ed68a3da14